docs/team/eugenetanwl3881.md

```diff
@@ -16,48 +16,46 @@ Given below are my contributions to the project.
 
   * What it does: allows the user to tag a valid item with a valid tag (PR [\#171](https://github.com/AY2223S1-CS2103T-W16-2/tp/pull/171), [\#172](https://github.com/AY2223S1-CS2103T-W16-2/tp/pull/172))
```

Comment:
Are PR links like these supposed to be changed to the new format as well?

Reply:
I'll leave it to the individual members to ensure it's formatted correctly. I just did a simple ctrl+f to fix all those that are already in the correct format as specified by the PPP checker (PR#XXX). Any other format, I'll leave it to them.